test/contquadratic.jl

```diff
@@ -1,144 +1,125 @@
 function set_qp1test_solutions!(solver)
     # SOLVE 1
-    LQOI.set_termination_status!(solver, MOI.Success)
-    LQOI.set_primal_status!(solver, MOI.FeasiblePoint)
-    LQOI.set_dual_status!(solver, MOI.FeasiblePoint)
-
-    LQOI.set_variable_primal_solution!(solver, [0.571429, 0.428572, 0.857143])
-    LQOI.set_constraint_primal_solution!(solver, [4.0, 1.0])
-    LQOI.set_quadratic_primal_solution!(solver, Float64[])
-
-    LQOI.set_variable_dual_solution!(solver, [0.0, 0.0, 0.0])
-    LQOI.set_constraint_dual_solution!(solver, [0.714286, 0.857144])
-    LQOI.set_quadratic_dual_solution!(solver, Float64[])
-
-    nothing
+    LQOI.set_solution!(solver,
+        termination_status = MOI.Success,
+        primal_status = MOI.FeasiblePoint,
+        dual_status = MOI.FeasiblePoint,

```

Comment:
Success and FeasiblePoint are defaults. There are lots of other tests with these and similar defaults that don't need to be given.